AirCasting/Utils/TextView.swift

```diff
@@ -20,13 +21,14 @@ struct TextView: UIViewRepresentable {
         textView.backgroundColor = UIColor.aircastingGray.withAlphaComponent(0.05)
         textView.layer.borderColor = UIColor.aircastingGray.withAlphaComponent(0.1).cgColor
         textView.layer.borderWidth = 1
+        textView.textColor = noteIsEditing ? .black : .lightGray
```

Comment:
Did you check if this "noteIsEditing" can be avoided by checking if text var is empty or not?

Reply:
I checked that but it doesn't work properly.